hugo.yaml

```diff
@@ -108,6 +108,7 @@ params:
 
   latest_engine_api_version: "1.47"
   docker_ce_version: "27.4.0"
+  docker_ce_version_prev: "27.3.1"
```

Comment:
Do you think we need to add comments to describe what they're used for?

Reply:
would probably be good to add more comments, generally. There's quite a bit of config here. Will do in a followup

Reply:
Maybe even restructure the config. It's all in one file now, but we could use a `config` directory with different files for params, general config, menus, etc.